src/SimpleAmqpClient/Channel.h

```diff
@@ -134,12 +137,14 @@ class SIMPLEAMQPCLIENT_EXPORT Channel : boost::noncopyable {
                             const std::string &password = "guest",
                             const std::string &vhost = "/",
                             int frame_max = 131072,
-                            bool verify_hostname = true) {
+                            bool verify_hostname = true,
+                            bool verify_peer = true) {
```

Comment:
This is going to cause unexpected results for users that have `verify_hostname = false` set.  Previous behavior is that both verifications are ignored, new behavior is that only the hostname verification is ignored (and this will all happen without a complier error or warning).

Likely the way options are passed in here needs to be changed to support this new use case.

Reply:
Indeed. I updated the PR to keep the previous behavior.